scripts/test.sh

```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
```

Comment:
use `-e` option so that the script stops if there are errors. Also `-u` option for unbound variables can be useful